src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -0,0 +1,142 @@
+                "sha1 varchar, feed_id varchar, feed_version varchar, filename varchar, loaded_date timestamp, " +
+                "snapshot_of varchar)");
+        connection.commit();
+        LOG.info("feeds table created");
+
+        // create an empty snapshot to create a new namespace and all the tables
+        FeedLoadResult result = makeSnapshot(null, testDataSource);
+        testNamespace = result.uniqueIdentifier;
+    }
+
+    @Test
+    public void canCreateUpdateAndDeleteFeedinfoEntities () throws IOException, SQLException {
+        ObjectMapper mapper = new ObjectMapper();
+
+        // create new object to be saved
+        FeedInfoDTO feedInfoInput = new FeedInfoDTO();
+        String publisherName = "test-publisher";
+        feedInfoInput.feed_publisher_name = publisherName;
+        feedInfoInput.feed_publisher_url = "example.com";
+        feedInfoInput.feed_lang = "en";
+        feedInfoInput.default_route_color = "1c8edb";
+        feedInfoInput.default_route_type = "3";
+
+        // convert object to json and save it
+        JdbcTableWriter createTableWriter = new JdbcTableWriter(Table.FEED_INFO, testDataSource, testNamespace);
+        String createOutput = createTableWriter.create(mapper.writeValueAsString(feedInfoInput), true);
+        LOG.info("create output:");
+        LOG.info(createOutput);
+
+        // parse output
+        FeedInfoDTO createdFeedInfo = mapper.readValue(createOutput, FeedInfoDTO.class);
+
+        // make sure saved data matches expected data
+        assertThat(createdFeedInfo.feed_publisher_name, equalTo(publisherName));
+
+        // try to update record
+        String updatedPublisherName = "test-publisher-updated";
+        createdFeedInfo.feed_publisher_name = updatedPublisherName;
+
+        // covert object to json and save it
+        JdbcTableWriter updateTableWriter = new JdbcTableWriter(Table.FEED_INFO, testDataSource, testNamespace);
+        String updateOutput = updateTableWriter.update(
+            createdFeedInfo.id,
+            mapper.writeValueAsString(createdFeedInfo),
+            true
+        );
+        LOG.info("update output:");
+        LOG.info(updateOutput);
+
+        FeedInfoDTO updatedFeedInfoDTO = mapper.readValue(updateOutput, FeedInfoDTO.class);
+
+        // make sure saved data matches expected data
+        assertThat(updatedFeedInfoDTO.feed_publisher_name, equalTo(updatedPublisherName));
+
+        // try to delete record
+        JdbcTableWriter deleteTableWriter = new JdbcTableWriter(Table.FEED_INFO, testDataSource, testNamespace);
+        int deleteOutput = deleteTableWriter.delete(
+            createdFeedInfo.id,
+            true
+        );
+        LOG.info("delete output:");
+        LOG.info(updateOutput);
+
+        // make sure record does not exist in DB
+        String sql = "select * from " + testNamespace + "." + Table.FEED_INFO.name +
```

Comment:
I've begun trying to use `String.format` more consistently in this codebase.  Could you replace instances where strings are built with `+`?

Reply:
Will do